pkg/plugins/policies/core/rules/merge/merge.go

```diff
@@ -363,7 +363,12 @@ func appendSlices(dst reflect.Value, src reflect.Value) {
 			if dstField.Elem().Kind() == reflect.Slice {
 				mergeByKey := strings.Contains(field.Tag.Get(policyMergeTag), mergeValuesByKey)
 				if strings.HasPrefix(field.Name, appendSlicesPrefix) || mergeByKey {
-					s := reflect.AppendSlice(dstField.Elem(), srcField.Elem())
+					var s reflect.Value
+					if srcField.Elem().Kind() == reflect.Invalid {
```

Comment:
probably it's better to check `srcField.IsNil()` rather then trying to dereference and check the result is invalid

Reply:
👍